python/py-ansible-core/Portfile

```diff
@@ -31,7 +31,7 @@ long_description \
     be written in any language and are transferred to managed machines \
     automatically.
 
-python.versions     37 38 39 310
```

Comment:
did you verify that no other ports depend anymore on the py37/py38 subports? If that's really the case you should remove the select files for these two versions as well.

Reply:
I did a quick grep:
```
❯ find . -name Portfile -exec egrep -H -m1 ansible {} \+
./python/py-ansible-compat/Portfile:name                py-ansible-compat
./python/py-ansible-core/Portfile:name                py-ansible-core
./python/py-ansible/Portfile:name                py-ansible
./python/ansible_select/Portfile:name                ansible_select
./sysutils/ansible-lint/Portfile:github.setup        ansible-community ansible-lint 6.9.1 v
./sysutils/ansible/Portfile:name            ansible
```

I checked `ansible-compat` and `ansible-lint`: they do not depend on removed subports of ansible